_posts/2022-04-19-towards-sustainable-storage.md

```diff
@@ -0,0 +1,39 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: Towards a sustainable storage, enabling co-financing of public infrastructure
+tags:
+- citations
+author: bgruening
+supporters:
+- crc1425
+- crc992
+- denbi
+- elixir
+---
+
+tl;dr: Research groups and networks can include their own storage into the Europan Galaxy server and decide about quota and data policies.
+
+<hr/>
+
+If people ask us what do they need to run a Galaxy service, one of our answers is
+"Storage is always the bottleneck. Try to get your hands on enough storage, this provides your users freedom to play around with their data."
+The European Galaxy server, thanks to the de.NBI cloud and the Uni-Freiburg compute center, manages 2PB of data today (2022/04).
+Users do use this storage for free, public governments are paying for the freedom of science and against being locked in into some commercial system.
+Everyone gets a fair share of the storage, on the [EU Galaxy server](https://usegalaxy.eu) those are 250GB for every user.
+This system works and has served us well for many years. However, different user groups have different needs and a
+few user groups provide funding to the global infrastructure in one way or the other. Our solution until now was to be
+very generous in extending the user storage via our quota-request form - and we will try to do so as much as we can increase our storage capacity with future grants.
+
+Going forward, we have [enabled](https://github.com/usegalaxy-eu/sorting-hat/pull/9/) an additional strategy that makes it possible to
+<b>include storage provided by partner consortia in a very transparent way into the European Galaxy Server</b> to increase quota to all consortia members.
```

Comment:
```suggestion
To address the special needs of user groups within a consortium, we have [enabled](https://github.com/usegalaxy-eu/sorting-hat/pull/9/) an additional strategy that makes it possible to
**include storage provided by their consortium in a very transparent way into the European Galaxy Server**, so that all the members get their quota automatically increased.
```

Reply:
(not sure how accurate this suggestion is)

Reply:
This shifts the focus to the user-groups, but not to the new functions, isn't it?

Reply:
hmm then I'm missing the new functions I guess, what's the difference?